verifier/src/lib.rs

```diff
@@ -2,14 +2,13 @@
 
 use air::{ProcessorAir, PublicInputs};
 use core::fmt;
-pub use vm_core::{Kernel, ProgramInfo, StackInputs};
-pub use winterfell::VerifierError;
+use vm_core::crypto::hash::Blake3_192;
 
 // EXPORTS
 // ================================================================================================
 
-pub use vm_core::{chiplets::hasher::Digest, StackOutputs, Word};
-pub use winterfell::StarkProof;
+pub use vm_core::{Kernel, ProgramInfo, StackInputs, StackOutputs, Word};
+pub use winter_verifier::{StarkProof, VerifierError};
```

Comment:
We should still export `Digest` from here so that the user is able to construct `Kernel` structs.

Reply:
Fixed!